migrations/1644469112765_create-slot-usage-cheques.ts

```diff
@@ -0,0 +1,28 @@
+import { MigrationBuilder } from 'node-pg-migrate'
+import { SlotUsageCheque } from '../src/SlotUsageCheque'
+
+const tableName = SlotUsageCheque.tableName
+
+export const up = (pgm: MigrationBuilder) => {
+  pgm.createTable(
+    tableName,
+    {
+      id: { type: 'UUID', primaryKey: true, unique: true, notNull: true },
+      signature: { type: 'TEXT', notNull: true },
+      quantity: { type: 'TEXT', notNull: true },
+      salt: { type: 'TEXT', notNull: true },
```

Comment:
Do we need the salt to be a column? Ins't it going to be part of the signed message?

Reply:
It is, but we decided on adding it here too when creating https://github.com/decentraland/builder-server/issues/390 with @nachomazzara . I can't really recall if it's necessary now